models/token_data.py

```diff
@@ -10,7 +10,7 @@
 class ReadConfig(BaseModel):
     """the ReadConfig class"""
 
-    config: dict[str, str] = load_config(Path("config/token.json"))
+    config: dict[str, str] = load_config(Path("config/token.json"))  # type: ignore
```

Comment:
linters complaining about typing for this line?

Reply:
yes

Reply:
weird on my machine linter not complaining? what's the message?